.travis.yml

```diff
@@ -4,3 +4,15 @@ go: "1.11"
 install: true
 script:
 - GINKGO_EXTRA_ARGS="--compilers=2" make all test/unit
+
+deploy:
+  - provider: script
+    script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make handler push-handler
```

Comment:
What is docker? Do you mean quay? ;)

Reply:
we are login to quay I just reuse the variable names from our other projects